bancor_research/bancor_simulator/v3/spec/network.py

```diff
@@ -99,11 +99,6 @@ def __init__(
             withdrawal_fee=toDecimal(withdrawal_fee),
         )
 
-        # initialize bnt
-        state.tokens["bnt"] = Tokens(tkn_name="bnt")
-        state.tokens["bnbnt"] = Tokens(tkn_name="bnbnt")
-        state.tokens["vbnt"] = Tokens(tkn_name="vbnt")
```

Comment:
It's not clear to me why these are being removed?

Reply:
Moved to [here](https://github.com/bancorprotocol/research/pull/79#discussion_r943726084), since otherwise, the BNT cooldown time remains the default (instead of being initialized to the specified value).

That's why I wrote in that comment, that it feels that we should have a single cooldown time for all tokens.
But I'm also OK with your Lamborghini argument.